### ISSUE TITLE
Midterm Review - ckb65

### ISSUE THREAD

**Issue by Caleb-berman** [pangolion/Mental-Project#12]
This group is investigating the preponderance of mental health issues in the workplace. They're doing this via survey data collected from questionnaires given out 2014-2019. The main objective of theirs is to predict whether an employee has sought treatment for a mental-health issue from a professional.

things I liked:

- They included a good, detailed analysis and plan of determining if their model was underfitting or overfitting.
- They utilized a wide set of models and regularizers to find the optimal loss function to fit to their data.
- They outlined many different plans of dealing with the large amount of missing data within their dataset and gave good rationale for why they chose replacing the missing data with the mean of that column
- The use of ordinal encoding was well rationalized and made sense for the data that they were using.

areas for improvement:

- The visualizations present are lacking in axis titles and readability. For example, your correlation matrix (Figure 7) is currently unreadable in the report as it's too zoomed out and dense.
- The report has a fair amount of grammatical / spelling mistakes that distract from the actual text of the report. (for example: "package" is spelled "packcage")
- In the end it was unclear exactly what features they used in generating their models. They mentioned that they have yet to encode the text-based features and give the total number of features but it'd be helpful to know exactly how many features and which ones they used in the models reporting to judge overfitting/under-fitting characteristics.
- It sounds like from what we recently learned in lecture using a GLRM to impute your missing values would be a really useful solution beyond just replacing missing values with the mean.
